src/network/bridge.rs

```diff
@@ -550,6 +569,11 @@ fn create_interfaces(
                     // for all other errors we want to return the error
                     return Err(err).wrap("get bridge interface");
                 }
+
+                if let BridgeMode::L2 = data.mode {
+                    return Err(err).wrap("l2 bridge interface not found");
+                }
```

Comment:
I wonder if user will be running into https://github.com/containers/netavark/commit/730e1bffb6e0bf0fb1f597195454e581ea9804be if we do not create the bridge. There seems to be bad things going on while/adding removing new interfaces.

I think the error here should say l2 requires the bridge to already exist on the host